org_fedora_oscap/content_handling.py

```diff
@@ -27,7 +27,10 @@
 import os.path
 
 from collections import namedtuple, OrderedDict
-from openscap_api import OSCAP
+try:
+    from openscap_api import OSCAP
+except ImportError:
+    import openscap as OSCAP
```

Comment:
This might be fixed when https://github.com/OpenSCAP/openscap/pull/810 is merged.